services/ext/service.go

```diff
@@ -567,7 +568,13 @@ func (s *Service) FetchCollectibleMetadata(id thirdparty.CollectibleUniqueID, to
 	}
 
 	// Try to fetch metadata from Messenger communities
-	community, err := s.messenger.RequestCommunityInfoFromMailserver(communityID, true)
+
+	// TODO: we need the shard information in the collectible to be able to retrieve info for
+	// communities that have specific shards
+
+	var shardCluster *uint // TODO: get this value from token
+	var shardIndex *uint   // TODO: get this value from token
```

Comment:
This requires smart contract work to obtain the shard cluster and index from the collectibles